### ISSUE TITLE
Permit --legacyAuth to NPM Plugin

### ISSUE THREAD

**Issue by klaygomes** [intuit/auto#1267]
**Is your feature request related to a problem? Please describe.**

All jfrog documentation is around using legacy auth.

**Describe the solution you'd like**

Include `--legacy-auth` option as lerna did https://github.com/agirton/lerna/blob/688f38fadb991874535483e73aad75277c6f865b/commands/publish/index.js#L119

**Describe alternatives you've considered**

**Additional context**

**Comment by hipstersmoothie** [comment]
@klaygomes could you try our this PR https://github.com/intuit/auto/pull/1268

`9.37.1-canary.1268.16194.0`

You'll need to change `auto` to `@auto-canary/auto` and all plugins from `@auto-it` to `@auto-canary`

**Comment by klaygomes** [comment]
It worked very well!
![image](https://user-images.githubusercontent.com/522884/83342105-7d222900-a2c1-11ea-833c-d9782942ba30.png)



For everyone trying to configure jFrog to use with Auto, below is how I achieve it:

![image](https://user-images.githubusercontent.com/522884/83342122-b064b800-a2c1-11ea-97dc-11fd6223add9.png)

On CI, I create a `.npmrc` as I'm used to:

```bash
$ curl -u${JFROG_USERNAME}:${JFROG_PASSWORD} ${JFROG_NPM_REPOSITORY}/auth/mycompany > .npmrc
```

Just before calling `auto` I'm reusing the auth provide by jFrog and exporting it as a `NPM_TOKEN` variable:

```bash
export NPM_TOKEN=$(cat .npmrc | cut -f 2 -d = | sed -n 2p)
```

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v9.38.0`](https://github.com/intuit/auto/releases/tag/v9.38.0) :rocket:
